src/quicksand/sand_handler.cr

```diff
@@ -23,8 +23,8 @@ class Quicksand::SandHandler
     when "/"
       context.response.status_code = 302
       context.response.headers["Location"] = "/" + file_path
-    when "/" + file_path
-      context.response.headers["Etag"] = %{W/"#{mod_time.epoch}"} if mod_time
+    when file_path
```

Comment:
This change causes the program to not work for me; is there a reason you made this change?

Reply:
It should work, let me test again. On my system it was adding an extra `/` which was messing things up.

Reply:
Yeah I just re-tested, with the `/` beforehand it doesn't work on either of the systems I've tested it on. Something that should work is if I do `File.join("/", file_path)`. I can push that update quickly.

Reply:
Awesome!